Src/Base/AMReX_MPMD.H

```diff
@@ -26,18 +26,27 @@ int MyProgId (); //! Program ID
 class Copier
 {
 public:
-    Copier (BoxArray const& ba, DistributionMapping const& dm);
+    Copier ();
```

Comment:
Default ctor is very special. So I think it's better to add a dummy argument (e.g., `bool`) to the ctor to make it non-default.